### PR TITLE
Add test suite and linter

### PR DESCRIPTION
**Pull request by romidane**
- Fix lint issues
- Add a couple dot files

### CODE REVIEW

File: log.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const bunyan = require('bunyan');
 const bunyanFormat = require('bunyan-format');
-const formatOut = bunyanFormat({outputMode: 'short'});
 
-const log = bunyan.createLogger({name: 'Starter app', stream: formatOut, level: 'debug'});
+const formatOut = bunyanFormat({ outputMode: 'short' });
```
Comment (whitfield-mj) [inline log.js:4]: You've put spaces after curly brackets #vomit

File: server.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
+require('dotenv').config();
+require('dotenv').config();
```
Comment (whitfield-mj) [inline server.js]: Twice?

Comment (romidane) [reply server.js]: oops 👍

File: log.js (hunk 1 above)
Comment (romidane) [reply log.js:4]: haha linting wars begin 😈

Comment (romidane) [reply log.js:4]: Users can override this in the linting rules? Just went with the standard AirBnb

Comment (whitfield-mj) [reply log.js:4]: I can live with it

**Review by whitfield-mj — APPROVED** [review]
Looks good

**Pull request merged**